src/core/amount.js

```diff
@@ -38,25 +35,25 @@ function Amount() {
 
 Amount.strict_mode = true;
 
-var consts = {
+const consts = {
   currency_xns: 0,
   currency_one: 1,
   xns_precision: 6,
 
   // bi_ prefix refers to "big integer"
   // TODO: we shouldn't expose our BigNumber library publicly
-  bi_5: new BigNumber(5),
-  bi_7: new BigNumber(7),
-  bi_10: new BigNumber(10),
-  bi_1e14: new BigNumber(1e14),
-  bi_1e16: new BigNumber(1e16),
-  bi_1e17: new BigNumber(1e17),
-  bi_1e32: new BigNumber(1e32),
-  bi_man_max_value: new BigNumber('9999999999999999'),
-  bi_man_min_value: new BigNumber(1e15),
-  bi_xns_max: new BigNumber(1e17),
-  bi_xns_min: new BigNumber(-1e17),
-  bi_xns_unit: new BigNumber(1e6),
+  bi_5: new Value(5)._value,
```

Comment:
I don't know if anyone is using these, but now might be a good time to delete them. We shouldn't be exposing the bignumber implementation here.